x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/badge/span_links_badge.tsx

```diff
@@ -47,7 +55,14 @@ export function SpanLinksBadge({ linkedParents, linkedChildren, id }: Props) {
         </EuiFlexGroup>
       }
     >
-      <EuiBadge data-test-subj={`spanLinksBadge_${id}`}>
+      <EuiBadge
+        data-test-subj={`spanLinksBadge_${id}`}
+        onClick={(e: any) => {
+          e.stopPropagation();
+          onClick(spanLinksFlyoutTab);
+        }}
+        onClickAriaLabel="Open span links details"
```

Comment:
Can you please add the translation for the aria-label

Reply:
fixed here